ARM-wvd-templates/HCI/QuickDeploy/AddHciVirtualMachinesQuickDeployTemplate.json

```diff
@@ -0,0 +1,283 @@
+        "metadata": {
+          "description": "Number of session hosts that will be created and added to the hostpool."
+        },
+        "defaultValue": 1
+      },
+      "vmNamePrefix": {
+        "type": "string",
+        "metadata": {
+          "description": "This prefix will be used in combination with the VM number to create the VM name. If using 'rdsh' as the prefix, VMs would be named 'rdsh-0', 'rdsh-1', etc. You should use a unique prefix to reduce name collisions in Active Directory."
+        }
+      },
+      "imageId": {
+        "type": "string",
+        "metadata": {
+          "description": "Id of gallery image on the Azure Stack HCI cluster."
+        }
+      },
+      "virtualNetworkId": {
+        "type": "string",
+        "metadata": {
+          "description": "Resource Id of the subnet used for the VMs"
+        }
+      },
+      "tags": {
+        "type": "object",
+        "metadata": {
+          "description": "The tags to be assigned to the virtual machines"
+        },
+        "defaultValue": {}
+      },
+      "deploymentId": {
+        "type": "string",
+        "metadata": {
+          "description": "GUID for the deployment"
+        },
+        "defaultValue": "[newGuid()]"
+      }
+    },
+    "variables": {
+      "azureStackHciTemplateUri": "https://raw.githubusercontent.com/Azure/RDS-Templates/master/ARM-wvd-templates/nestedtemplates/azurestackhci-vm.json",
+      "rdshPrefix": "[concat(parameters('vmNamePrefix'),'-')]",
+      "rdshVmNamesOutput": {
+        "copy": [
+          {
+            "name": "rdshVmNamesCopy",
+            "count": "[parameters('vmNumberOfInstances')]",
+            "input": {
+              "name": "[concat(variables('rdshPrefix'), add(parameters('vmInitialNumber'), copyIndex('rdshVmNamesCopy')))]"
+            }
+          }
+        ]
+      }
+    },
+    "resources": [
+      {
+        "apiVersion": "2018-05-01",
+        "name": "[concat('UpdateHostPool-', parameters('deploymentId'))]",
+        "type": "Microsoft.Resources/deployments",
+        "resourceGroup": "[parameters('hostpoolResourceGroup')]",
+        "condition": "[not(empty(parameters('hostpoolResourceGroup')))]",
+        "properties": {
+          "mode": "Incremental",
+          "template": {
+            "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+            "contentVersion": "1.0.0.0",
+            "resources": [
+              {
+                "name": "[parameters('hostPoolName')]",
+                "apiVersion": "2019-12-10-preview",
+                "location": "[parameters('hostpoolLocation')]",
+                "type": "Microsoft.DesktopVirtualization/hostpools",
+                "properties": "[parameters('hostpoolProperties')]"
+              }
+            ]
+          }
+        }
+      },
+      {
+        "apiVersion": "2018-05-01",
+        "name": "[concat('hci-vmCreation-linkedTemplate-', parameters('deploymentId'))]",
```

Comment:
Why the name is create here instead of add

Reply:
same as other comment